styles/05-objects/card.css

```diff
@@ -0,0 +1,163 @@
+/* .cards, */
+
+.card-container,
+.card-primary-content,
+.actions {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.actions {
+  flex-wrap: nowrap;
+}
+
+.cards {
+  padding: 10px;
+  justify-content: space-around;
+  gap: 4rem;
+}
+
+.card-container {
+  background-color: white;
+  box-shadow: 0 0 5px rgb(58, 57, 58);
+  row-gap: 5px;
+  width: 400px;
+  flex-direction: column;
+  border-radius: 15px 15px 20px 20px;
+}
+
+.card-primary-content {
+  flex-direction: column;
+}
+
+.card-primary-content:hover {
+  filter: grayscale(0.1) invert(0.1);
+  background-color: #fff;
+  backdrop-filter: contrast(100%);
+  transform: 0.3s;
+}
+
+.card-header {
+  margin: 1rem 0 0.5rem
+}
+
+.card-header,
+.card-body {
+  padding: 0 15px;
+}
+
+.decription {
```

Comment:
Description is a generic name. If this is a generic text related selector, please move to typography.css

Reply:
Had renamed it, forgot to change in css. Thanks!